big_tests/test.config

```diff
@@ -193,7 +193,9 @@
       {auth_method, "odbc"},
       {odbc_pools, "{pool, odbc, default}."},
       {odbc_server, "{odbc_server, {pgsql, \"localhost\", \"ejabberd\", \"ejabberd\", \"mongooseim_secret\",
-                                     [{ssl, required}, {ssl_opts, [{verify, verify_peer}, {cacertfile, \"priv/ssl/cacert.pem\"}]}]}}."},
+                                     [{ssl, required}, {ssl_opts, [{verify, verify_peer},
+                                                                   {cacertfile, \"priv/ssl/cacert.pem\"},
+                                                                   {server_name_indication, disable}]}]}}."},
```

Comment:
Why is it needed for OTP 21?

Reply:
Actually I was surprised it worked on 20. Basically since Erlang 20 (probably not 20.0 which we used in our tests) ssl client started by default verifying hostname (of the connection) with the name in the cert. In our tests this didn't match because we were connecting to `localhost` while the name in he cert was `MongooseIM`. This change is only for tests, with proper certs and correct hostnames this is no needed.